spec/integration_spec.cr

```diff
@@ -33,6 +34,7 @@ describe LuckyRouter do
       router.match!("get", "/users/1/new").payload.should eq :new
       router.match!("get", "/users/1/tasks/1").payload.should eq :user_tasks
       router.match!("get", "/admin/users/1/tasks/1").payload.should eq :admin_user_tasks
+      router.match!("get", "/admin/reports/1.csv").payload.should eq :admin_reports
```

Comment:
I'm worried that the param name key will be `"report_id.csv"` and not just `report_id`. Could you add a spec that makes sure the param payload is correct?

Reply:
Yup. good call. It would be `report_id.csv`. So looks like I may need to update some actual code then 😝